tcp_mux.go

```diff
@@ -109,33 +110,34 @@ func (m *TCPMuxDefault) LocalAddr() net.Addr {
 }
 
 // GetConnByUfrag retrieves an existing or creates a new net.PacketConn.
-func (m *TCPMuxDefault) GetConnByUfrag(ufrag string) (net.PacketConn, error) {
+func (m *TCPMuxDefault) GetConnByUfrag(ufrag string, forAddress net.IP) (net.PacketConn, error) {
```

Comment:
It seems the only thing we need to know here is whether the IP is v4 or v6. Did you have a special reason for passing the whole `net.IP` rather than just the `isIPv6 boolean` value here?

Reply:
@jeremija fixed!

Reply:
Thanks!